src/DependencyInjection/Configuration.php

```
@@ -181,6 +182,11 @@ private function addChromiumPdfOptionsNode(ArrayNodeDefinition $parent): void
                     ->info('Define whether to print the entire content in one single page. - default false. https://gotenberg.dev/docs/routes#page-properties-chromium')
                     ->defaultNull()
                 ->end()
+                ->enumNode('paper_size')
+                    ->info('Paper width and paper height, in inches - default 8.5 and 11. https://gotenberg.dev/docs/routes#page-properties-chromium')
```

Is this configuration compatible with `paper_width` and `paper_height` ? We could throw an exception, warn or, at least, document it.

---

Also, the value must be a `PaperSize` size value, not inches.